client/components/Header.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes, Component } from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+class Header extends Component {
+
+  handleClick(e) {
+    e.preventDefault();
+    this.props.onClick();
+  }
+
+  render() {
+    return (
+      <header>
+        <div className='grid'>
+          <div className='col-2-12'>
+            <object type='image/svg+xml' className='ninja' data='./../assets/ninja.svg' />
+          </div>
+          <div className='col-10-12'>
+            <h1>SCENIC<span className='black'>.NINJA</span></h1>
+            <p className='description'>After years spent scouring Google Places, Scenic Ninja is ready to share his wisdom with you.</p>
+            <button
+            onClick={ this.handleClick.bind(this) }
+            type="submit">Find Views
+            </button>
+            <a className='logout' href='/auth/logout'>Logout</a>
```

Comment:
Look into refactoring this to a form element with the submit action on the form element, with the button inside it. cc @reinaisnothere